### PR TITLE
Community map for Node-Network Page

### PR DESCRIPTION
**Pull request by jramos0**

### CODE REVIEW

File: packages/service-content/src/lib/resources/services/builders-locations.sync.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    })),
+);
+
+const fetchBuilderLocation = async (query: string) => {
+  const q = encodeURIComponent(query);
+
+  const res = await fetch(
+    `https://nominatim.openstreetmap.org/search?format=jsonv2&limit=1&q=${q}`,
+  );
+  console.log('Buscando ubicación para:', query);
```
Comment (max-lt) [inline packages/service-content/src/lib/resources/services/builders-locations.sync.ts]: Hello, english please ;)

File: packages/service-content/src/lib/resources/services/builders-locations.get.ts
Comment (max-lt) [inline packages/service-content/src/lib/resources/services/builders-locations.get.ts:1]: Very good

**Review by kevinmulier — APPROVED** [review]
Front -> OK

File: packages/service-content/src/lib/resources/services/builders-locations.sync.ts (hunk 1 above)
Comment (jramos0) [reply packages/service-content/src/lib/resources/services/builders-locations.sync.ts]: Hi Maxi, I’ve already fixed it. 
Thank you very much for the review ;)

**Pull request merged**